core/src/test/java/de/danoeh/antennapod/core/util/QueueSorterTest.java

```diff
@@ -0,0 +1,156 @@
+package de.danoeh.antennapod.core.util;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+
+import de.danoeh.antennapod.core.feed.Feed;
+import de.danoeh.antennapod.core.feed.FeedItem;
+import de.danoeh.antennapod.core.feed.FeedMedia;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertNull;
+
+/**
+ * Test class for QueueSorter.
+ */
+public class QueueSorterTest {
```

Comment:
Awesome! Adding unit tests is always a good idea :+1:

Reply:
Thanks :-)